src/main/java/com/example/springboot/controller/HelloController.java

```diff
@@ -1,24 +1,48 @@
 package com.example.springboot.controller;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 import com.example.springboot.ShoppingListService;
+import com.example.springboot.service.Product;
 
 @RestController
+@RequestMapping("/api/v1/item")
 public class HelloController {
+	List<Product> ls = new ArrayList<Product>();
```

Comment:
Data shouldn't be at controller level. Move it to the service (temporarily) until the database is ready